jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -214,7 +214,6 @@ protected void addColumnSortHandler(ExtendedPagedTable listTable) {
                     gridPreferencesStore.setGridSortedColumnPreference(gridSortedColumnPreference);
                 }
             }
-            listTable.saveGridPreferences();
```

Comment:
@bxf12315 sounds like we would persist the changes in the sorting at all with this change. I believe that might still be needed, @nmirasch wdyt?

Reply:
@cristianonicolai yes, sort column need persist, but it saves it in to current GridPreferencesStore, if create a new filter, it will save again. So If we delete it, do not impact save sort column into new filter.